src/constant.h

```diff
@@ -7,594 +7,598 @@
 // Global constants used by Moria
 // clang-format off
 
-#define MAX_UCHAR         255
-#define MAX_SHORT       32767   // maximum short/long signed ints
-#define MAX_LONG   0x7FFFFFFFL
+#pragma once
+
+#include "headers.h"
```

Comment:
This header is actually included in `headers.h` already, so we should be able to remove this pragma and include.